public_records_portal/upload_helpers.py

```diff
@@ -234,7 +257,14 @@ def upload_file_locally(document, filename, privacy, request_id = None):
 
 ### @export "allowed_file"
 def allowed_file(file, request_id, privacy=1):
-    mimetypeMagic = magic.Magic(mime=True,uncompress=True)
-    mimetype = mimetypeMagic.from_buffer(file.read())
+    '''Checks the mimetype of the file to see if its allowed'''
```

Comment:
Please update the docstring to include a description of the parameters and return type.